From 3e8ec9c43191f45057598aee2aabc2a1c0675ab8 Mon Sep 17 00:00:00 2001
From: utlark <utlark@ya.ru>
Date: Fri, 2 Sep 2022 00:47:55 +0500
Subject: [PATCH] This attribute generated many warnings

---
 include/linux/log2.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/log2.h b/include/linux/log2.h
index fd7ff3d9..14458fdd 100644
--- a/include/linux/log2.h
+++ b/include/linux/log2.h
@@ -18,7 +18,7 @@
 /*
  * deal with unrepresentable constant logarithms
  */
-extern __attribute__((const, noreturn))
+extern __attribute__((const))
 int ____ilog2_NaN(void);
 
 /*
-- 
2.34.1

